pkg/minikube/constants/constants.go

```diff
@@ -121,8 +121,10 @@ const (
 	DefaultKeepContext = false
 	// SHASuffix is the suffix of a SHA-256 checksum file
 	SHASuffix = ".sha256"
-	// DefaultMemory is the default memory of a host, in megabytes
-	DefaultMemory = 2048
+	// DefaultMemory is the default memory which will be allocated to minikube, parseable
+	DefaultMemory = "2048mb"
+	//MinimumMemorySizeMB is the minimum memory size, in megabytes
+	MinimumMemorySizeMB = 1500
```

Comment:
lets change this to 1200

Reply:
lets warn the user if they are less than DefaultMemory (2048mb)

Reply:
This is done. If the memory is now less than the default memory, a warning will be logged if the RAM is less than the Default Memory. If it is less than 1024, it will exit.